app/design/frontend/base/default/template/ebanx/form/pagofacil.phtml

```diff
@@ -0,0 +1,25 @@
+<?php
+    $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-rapipago.png" heigth="30" /></div>';
+?>
+
+<?= !$showInlineIcon ? null : $icon?>
```

Comment:
This is reaaally weird... "not (show) ? null : show" instead of "show ? show : null" bugs me